docs/concept-exercises.md

```diff
@@ -58,14 +58,14 @@ An exercise has the following files. In the browser they will show at the releva
 **Purpose:** Introduce the concept(s) that the exercise teaches to the student.
 
 - The information provided should give the student just enough context to figure out the solution themselves.
-- Links should be used sparingly if at all. While a link explaining a complex topic like recursion might be useful, for most concepts the links will provide more information than needed so explaining things concisely inline should be the aim.
-- The file should not give the student any extra information about a concept not needed to understand the fundamentals of the concept or solve the exercise.
-- Use the proper technical terms so that the student can easily search for more information.
-- If the exercise introduces new syntax, an example of the syntax should always be included; students should not need to search the web for examples of syntax.
+- Don't give any extra information about a concept not needed to understand the fundamentals of the concept or solve the exercise.
+- Links should be used sparingly, if at all. While a link explaining a complex topic like recursion might be useful, for most concepts the links will provide more information than needed so explaining things concisely inline should be the aim.
+- Use the proper technical terms (e.g. a keyword) so that the student can easily search for more information.
+- Don't use code examples unless the exercise introduces new syntax, in which case an example of the syntax must be included; students should not need to search the web for examples of syntax.
```

Comment:
```suggestion
- Code examples should only be used to introduces new syntax (students should not need to search the web for examples of syntax). In other cases provide descriptions or links instead of code.
```